.gitignore

```diff
@@ -1 +1,3 @@
 bin/
+vendor/
```

Comment:
I think we need to manage files under `vendor/*` so that users can import and use our code successfully.